src/jc_test.h

```diff
@@ -1170,6 +1171,7 @@ void jc_test_logf(const jc_test_fixture* fixture, const jc_test_entry* test, con
     }
     buffer[sizeof(buffer)-1] = 0;
     printf("%s", buffer);
+    fflush(stdout);
```

Comment:
On Github CI, we noticed that the jctest output arrived much later than the actual internal output.